src/Prima/Pyxis/Form/Example/Update.elm

```diff
@@ -267,3 +283,31 @@ updateUserPrivacyMarketing value =
 updateUserPrivacyThirdPart : Bool -> Model -> Model
 updateUserPrivacyThirdPart value =
     updateFormData (\f -> { f | userPrivacyThirdPart = Just value })
+
+
+applyAutocompleteCountryFilter : Maybe String -> ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
+applyAutocompleteCountryFilter filter ( model, cmd ) =
+    case filter of
+        Just value ->
+            ( model
+            , Cmd.batch [ cmd, fakeCallServerForAutocomplete value ]
+            )
+
+        Nothing ->
+            ( model, cmd )
+
+
+fakeCallServerForAutocomplete : String -> Cmd Msg
+fakeCallServerForAutocomplete filter =
+    List.filter (\( slug, _ ) -> String.contains filter slug)
```

Comment:
Questo è proprio il caso che ti dicevo, se io volessi utilizzare una lista statica sarei costretto a "mascherarla" da chiamata http inutilmente

Reply:
In realtà ho fatto quell'esempio sempre con l'idea che l'utilizzo dell'autocomplete fosse soprattutto con chiamate server,  quindi per facilitare chi lo avrebbe poi usato. Ma come hai già detto il consumer ha già tutti i dati per capire quando fare il filtro, quindi comunque ci sarà un approccio più semplice.

Reply:
questa non è una "chiamata mascherata"....è una funzione...è molto ideomatico usare una HOF in programmazione funzionale...non ci vedo niente di particolarmente brutto...